sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallAutomationAsyncClientAutomatedLiveTests.java

```diff
@@ -199,6 +200,70 @@ public void createVOIPCallAndRejectAutomatedTest(HttpClient httpClient) {
         }
     }
 
+    @ParameterizedTest
```

Comment:
We need to record test and update the asset? https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/35117/How-To-Run-Automated-SDK-E2E-Test?anchor=how-to-run-e2e-sdk-record-tests-locally-(java)

Reply:
Yes I am just waiting for the change to make it to production, so I will be updating the recording at that time thanks! Just wanted to get a PR review early.

Reply:
@v-vdharmaraj  I have updated the recording, let me know if it looks good to you!